plugin/core/signature_help.py

```diff
@@ -171,31 +169,30 @@ def __init__(self, signatures: 'List[SignatureInformation]',
     def build_popup_content(self, renderer: ScopeRenderer) -> str:
         parameter_documentation = None  # type: Optional[str]
 
-        formatted = []
+        formatted = ""
```

Comment:
joining a list of strings is "significantly faster": https://stackoverflow.com/questions/3055477/how-slow-is-pythons-string-concatenation-vs-str-join

Reply:
https://stackoverflow.com/a/3055558/8993294 shows the opposit.

Reply:
Let's ignore the stackoverflow crowd and do the experiment ourselves. In my Sublime Text console I do  the following:

```
>>> from timeit import timeit as f
>>> f('x="".join([str("asdfasdfasdf") for i in range(0, 100)])')
17.87444483214216
>>> f('x="".join(["asdfasdfasdf" for i in range(0, 100)])')
4.023197690061409
>>> f('x=""\nfor i in range(0, 100): x += "asdfasdfasdf"')
13.988012291068856
>>> f('x=""\nfor i in range(0, 100): x += str("asdfasdfasdf")')
28.351181478097544
```
The `"".join([])` variant seems fastest. However, it becomes slower once we use a call to `str`. I'm not sure why. The `+=` variant also becomes slower when wrapping it in `str`.

Reply:
Ok I reverted that in f0587379784f496d017022a86223f4c189a3f82e